spec/lib/ecco/row_event_listener_spec.rb

```diff
@@ -26,7 +26,7 @@
       subject.on_event(table_event)
       subject.on_event(row_event)
 
-      expect(actual_type).to eq(row_event_type)
+      expect(actual_type).to eq("WRITE_ROWS")
```

Comment:
Will try to make this more generic, or more specific that it's a write row event (now it's "hidden" in the spec context)